src/Controls/src/Core/Platform/iOS/Extensions/ViewExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using UIKit;
+
+namespace Microsoft.Maui.Controls.Platform;
+
+static class UIViewExtensions
+{
+	public static void AdjustBounds(UIView? childView, UIView? superView)
```

Comment:
Maybe use it as an extension method? 

```suggestion
	public static void AdjustBounds(this UIView? childView, UIView? superView)
```

Also, maybe we need a slightly better name to indicate what this will do. Maybe `SetFrameToFillSuperview` or `SetFrameToParentBounds`. I am sorry, I am not good at naming. @PureWeen I know you have the best names, so you have any ideas.

Reply:
Maybe this should be on the view controller instead? `void FitSubViewToAvailableSpace(this UIViewController vc)` so then we don't need to duplicate the vc.View and then pass 2 args.

Reply:
FitToAvailableSpace(this UIView, UIViewcontroller)

I think passing both in is important. 
The VC might have multiple subviews so we can just fit them all

Reply:
@PureWeen so should do a foreach on Subviews or some kind of recursion?